index.html

```diff
@@ -44,5 +44,160 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
             </div>
         </div>   
     </header> 
+    <!-- projects section -->
+    <section class="projects-container">
+        <div class="project">
+            <img src="./images/project1.png" class="project-img" alt="">
+            <div class="project-txt">
+                <h1 class="project-title">Tonic</h1>
```

Comment:
- [ ] You should use only one `<h1>` in an HTML page for semantic reasons. This will help for SEO and screen readers. It has to be assigned to the most important heading in the page.
```
  h1
    h2
      h3
      h3
        h4
        h4
          h5
          h5
            h6
    h2
```